webpack.config.js

```diff
@@ -55,11 +55,16 @@ module.exports = {
           //https://glenmaddern.com/articles/css-modules
           //https://github.com/css-modules/css-modules
             {
-                test: [/\.css$/, /\.scss$/, /\.sass$/],
-                loader: ExtractTextPlugin.extract({
-                    fallback: 'style-loader',
-                    use: 'style-loader!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!sass-loader!postcss-loader'
-                })
+                test: /\.css$/,
+                loader: 'style-loader'
+            }, 
+            {
+                test: /\.css$/,
+                loader: 'css-loader',
```

Comment:
Also might want to save the extractTextPlugin version for when we create a production webpack config. It outputs the css into a loadable static file as opposed to baking it into the js like the default loader does. Should boost performance [documentation](https://github.com/webpack-contrib/extract-text-webpack-plugin).

The normal loader does make it easier to enable [hot module reloading](http://matthewlehner.net/react-hot-module-replacement-with-webpack/) which is dank af